Framework/Kernel/src/PropertyNexus.cpp

```diff
@@ -108,6 +108,107 @@ std::unique_ptr<Property> makeStringProperty(::NeXus::File *file, const std::str
   }
 }
 
+//----------------------------------------------------------------------------------------------
+/** Opens a NXlog group in a nexus file and
+ * creates the correct Property object from it. Overlead that
+ *
+ * @param file :: NXS file handle
+ * @param group :: name of NXlog group to open
+ * @param fileDescriptor :: HDF5 Nexus file descriptor (metadata)
+ * @param prefix :: current absolute target prefix
+ * @return Property pointer
+ */
+std::unique_ptr<Property> loadProperty(::NeXus::File *file, const std::string &group,
```

Comment:
Can we factor our portions of this member function that are the same as in `std::unique_ptr<Property> loadProperty(::NeXus::File *file, const std::string &group)`?